summarization/summarizer_main.py

```diff
@@ -301,6 +321,7 @@ def _assemble_and_instantiate_stamp_page(
             sentence_in_para_index,
             sentence_in_para_weight,
             is_embedded_content,
+            is_quoted_content,
```

Comment:
Ideally, stamppage type should be enum. We should introduce lots of booleans, when we have only one type.

Reply:
StampPageType

Reply:
Yeah, we should do a code clean up once, I think we use strings in some places. I think now that code is checked in, we can take a high level look and do some refactorings.